creusot-contracts/src/std/ops.rs

```diff
@@ -254,3 +261,27 @@ extern_spec! {
         }
     }
 }
+
+#[cfg(not(creusot))]
+pub trait FnOnceExt<Args> {
+    type Output;
+}
+
+#[cfg(not(creusot))]
+pub trait FnMutExt<Args>: FnOnceExt<Args> {}
+
+/// `FnExt` is an extension trait for the `Fn` trait, used for
+/// adding a specification to closures. It should not be used directly.
+#[cfg(not(creusot))]
+pub trait FnExt<Args>: FnMutExt<Args> {}
+
+#[cfg(not(creusot))]
+impl<A, O, F: FnOnce(A) -> O> FnOnceExt<(A,)> for F {
+    type Output = O;
+}
+
+#[cfg(not(creusot))]
+impl<A, O, F: FnMut(A) -> O> FnMutExt<(A,)> for F {}
+
+#[cfg(not(creusot))]
+impl<A, O, F: Fn(A) -> O> FnExt<(A,)> for F {}
```

Comment:
TODO: finish this hack or find another replacement (this change is because the `Tuple` and `Fn<_>` traits are unstable).

Reply:
We should keep them for the same reason!